db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Retirement/StateMachines/Methods.class/__methods__/microsoft_check_pre_retirement.rb

```diff
@@ -0,0 +1,24 @@
+#
+# Description: This method checks to see if the azure instance has been powered off
+#
+
+# Get vm from root object
+vm = $evm.root['vm']
+
+if vm
+  power_state = vm.attributes['power_state']
```

Comment:
I know this is used in the other pre_retirement methods but it is an old syntax.  It should be: `vm.power_state`.  We should update all the retirement calls like this in a follow-up PR.